### ISSUE TITLE
hard to switch between dirrectories

### ISSUE THREAD

**Issue by babunadze** [ashkulz/NppFTP#263]
### Description of the Issue

hard to switch between dirrectories
### Steps to Reproduce the Issue

1. add profile
2. open folder
3. switch to another folder

### Expected Behavior

it will be better if you add filters (shotcurts) for dirrectories like in zend studio, 
it is very good feature i am staing on zend studio because this feature only

**Comment by chcg** [comment]
@babunadze Maybe as workaround, if you have a limited amount of folders, you could use multiple profiles for the same server and add there different initial remote dirs.

**Comment by babunadze** [comment]
>  @babunadze Maybe as workaround, if you have a limited amount of folders, you could use 
> multiple profiles for the same server and add there different initial remote dirs.

good idea, thank you, but i am working with multiple projects in each project about 10-15 folders,
also when folders are in many subfolders, orientation in left panel is very hard it shows full range of parent folders witch is not necessary.
